src/integration-tests/src/workflows.rs

```diff
@@ -155,14 +150,8 @@ main:
         .set_workflow(
             wf::model::Workflow::default()
                 .set_labels([("integration-test", "true")])
-                .set_call_log_level(
-                    wf::model::workflow::CallLogLevel::default()
-                        .set_value(wf::model::workflow::call_log_level::LOG_ERRORS_ONLY),
-                )
-                .set_state(
-                    wf::model::workflow::State::default()
-                        .set_value(wf::model::workflow::state::UNAVAILABLE),
-                )
+                .set_call_log_level(CallLogLevel::new("LOG_ERRORS_ONLY"))
+                .set_state(wf::model::workflow::State::new("UNAVAILABLE"))
```

Comment:
I would prefer:

```suggestion
                .set_call_log_level(wf::model::call_log_level::LOG_ERRORS_ONLY)
                .set_state(wf::model::workflow::state::UNAVAILABLE)
```

or something similar, is that possible?  We want to use the constants to avoid typos.

Reply:
Good point, done.